misc/test_expected_payload_cooker.py

```diff
@@ -0,0 +1,197 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+r"""
+This script helps to fix old/legacy payloads in libparsec serialization tests.
+
+Serialization tests are usually written in the following structure:
+- a hard-coded raw value (the payload)
+- a hard-coded expected value corresponding to the payload
+- an assert comparing the value loaded from raw to the expected value
+
+Any change impacting the serialization format or the expected value will make
+the payload obsolete. This scripts helps regenerate it for the failing test.
+
+1. Add a println! for the (dumped) expected value prefixed with "***expected: "
+
+   println!("***expected: {:?}", expected.dump());
+
+2. Run the failing test(s) and check that the print is displayed
+
+3. Run the failing test(s) redirecting stderr to stdout and pipe its output
+   int this script
+
+   cargo nextest run -p libparsec_protocol 2>&1 | python ./misc/test_expected_payload_cooker.py
```

Comment:
Maybe show an example of what's printed by the script?

Reply:
Done